static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -412,28 +429,59 @@ export default class AudioPlayer extends Component<Props, State> {
         }
     };
 
+    seekWave = (e: any) => {
+        if (document.querySelector('.fake-wave')) {
+            // $FlowFixMe
+            const boxW = document.querySelector('.fake-wave').offsetWidth;
+            const svg = document.querySelector('.fake-wave svg');
+            // $FlowFixMe
+            const pt = svg.createSVGPoint();
+
+            pt.x = e.clientX;
+            pt.y = e.clientY;
+            const adjustedPosition = pt.matrixTransform(
```

Comment:
🤯

I'd flag this as something that needs to be removed in a future iteration, we don't want to mess with coordinate systems.

Reply:
Sure. This code should be junked with the rest of the fake wave anyway